mmdet/datasets/api_wrappers/coco_api.py

```diff
@@ -0,0 +1,33 @@
+# This file add snack case alias for coco api
+
+from pycocotools.coco import COCO as _COCO
+from pycocotools.cocoeval import COCOeval as _COCOeval
+
+
+class COCO(_COCO):
+
```

Comment:
May add a docstring to indicate the difference between _COCO and COCO and our motivation.